book/projects/index.md

```diff
@@ -1,29 +1,29 @@
-# Projects
-
-During hackweeks we invite participants to collaborate in small group project teams (usually 5-7 people). Projects provide unique opportunities for networking, advancing research and learning new things:
-
-```{image} ../img/project-purpose.png
-:alt: diagram of the purpose of hackwek project work
-:class: bg-primary mb-1
-:width: 600px
-:align: center
-```
-
-We encourage you to be creative in how you design your project work! Here are some ways people have used project time in the past:
-
-
-* *focus on learning*: dive deeper into tutorial content and gain more hands on experience with new tools
-* *focus on community building*: hackweeks bring people together for the first time and projects are used to build social fabric and foster open science
-* *focus on research*: people often work to advance a research challenge using tools shared during the event. This often includes ongoing graduate and postgraduate investigations.
-* *focus on creating new tools*: hackweeks often expose general community software needs. Projects can be the catalyst for new community libraries.
-
-```{image} ../img/project-work.png
-:alt: picture of people interacting during hackweek project work
-:class: bg-primary mb-1
-:width: 500px
-:align: center
-```
-
-## Projects from Past Events
-
-Visit our [gallery of past hackweeks](https://uwhackweek.github.io/hackweeks-as-a-service/gallery/index.html) to browse our catalog of amazing projects from previous events!
+# Projects
+
+During hackweeks we invite participants to collaborate in small group project teams (usually 5-7 people). Projects provide unique opportunities for networking, advancing research and learning new things:
+
+```{image} ../img/project-purpose.png
+:alt: diagram of the purpose of hackwek project work
+:class: bg-primary mb-1
+:width: 600px
+:align: center
+```
+
+We encourage you to be creative in how you design your project work! Here are some ways people have used project time in the past:
+
+
+* *focus on learning*: dive deeper into tutorial content and gain more hands on experience with new tools
+* *focus on community building*: hackweeks bring people together for the first time and projects are used to build social fabric and foster open science
+* *focus on research*: people often work to advance a research challenge using tools shared during the event. This often includes ongoing graduate and postgraduate investigations.
+* *focus on creating new tools*: hackweeks often expose general community software needs. Projects can be the catalyst for new community libraries.
+
+```{image} ../img/project-work.png
+:alt: picture of people interacting during hackweek project work
+:class: bg-primary mb-1
+:width: 500px
+:align: center
+```
+
+## Projects from Past Events
+
+Visit our [gallery of past hackweeks](https://guidebook.hackweek.io/hackweeks-as-a-service/gallery/index.html) to browse our catalog of amazing projects from previous events!
```

Comment:
This link is a 404. the problem with the [escience version](https://escience.washington.edu/using-data-science/hackweeks/archive/) is it doesn't yet include the ICESat-2 Hackweeks...

Reply:
Another great find and thorough check of the links :+1: ... Fixed